### ISSUE TITLE
[review] Law of mass action and theory background

### ISSUE THREAD

**Issue by ppxasjsm** [alchemistry/alchemical-best-practices#103]
(1) In Chap 3.1 Eq suggests that the law of mass action can be applied to calculate K_b and Delta G_B from MD simulations. I am aware the authors just mean to provide the conceptual background here, but I am similarly aware of a few groups that actually have used the LMA. This is very dangerous. As shown in [54], the LMA does not strictly hold for systems with very few particles, and it contradicts in fact the more fundamental and general eq  (7). E.g., if a simulation system just contains one L and one R molecule, and these bind and unbind many times during the simulation, eq (7) would (correctly) predict that \Delta G = -kT log Z(RL)/Z(R+L) \approx -kT log T(RL)/T(R+L), where the latter means the time fraction of the trajectory where the system is bound / unbound. In contrast, the LMA would (wrongly) claim that Delta G = -kT log [RL]/[R][L] \approx -kT log T(RL)/(T(R+L))^2. The authors might want to add a word of caution to this not so well-known fact.

(2) Eq (4) is a bit confusing, at least to a physicist. Naively, I would say it would be correct without the concentration term: \Delta G then simply means the free energy difference for the concentration used in the simulation. Apparently, this is not meant here, but then I would refer to Delta G as the standard free energy, \Delta G^0.

(3) In Sec 3.2, the issue of a Delta G (Delta Delta G) between states with different number of atoms is touched upon. This issue is more subtle than the text suggests, and the authors might want to spend a few more sentences to explain under which conditions the ‘unit-of-volume-factor’ actually cancels out and when not. There’s chances for pitfalls for new practitioners!

**Comment by andrrizzi** [comment]
I've updated Sections 3.1 and 3.2 on the overleaf document to address (1) and (2).

For (3), can someone with more experience on relative calculations check my interpretation below?

I assume the reviewer refers to Eq. 9 and the last sentences at the end of that paragraph. I can see two things they are suggesting us to warn the user about:
1. If the dummy atoms bonding terms are removed in the noninteracting state, and the receptor/solvent boxes have different volumes, the contribution would not cancel out.
2. If a beginner would simply simulate the end states without dummy atoms, the free energy difference estimators are not applicable.
In both cases, I don't think these observations belong to the theory introduction section, but we could mention them, for example, in section 7, and just refer to those sections for more info.

**Comment by andrrizzi** [comment]
Possible answers for the response letter for (1) and (2):

(1) The referee raises a valid concern, and we have made the warning explicit in Section 3.2.
(2) The standard free energy is indeed what we are referring to throughout the paper. We have adopted the convention of using \Delta G (without the ^o) to refer to the standard free energy. This is common practice in the alchemical calculations literature as the standard free energy is typically the only quantity of interest and the one reported and compared to experiments. We agree this might create some confusion, and we have thus added a comment in Section 3.1 to clarify the notation.

**Comment by ppxasjsm** [comment]
@andrrizzi, can you add comments for your edits to the review document? https://www.overleaf.com/project/5c5879b06343590b46b70041

**Comment by andrrizzi** [comment]
Added answers for points (1) and (2) to the review document.

**Comment by ppxasjsm** [comment]
Thank you!

**Comment by andrrizzi** [comment]
Just to make sure this wasn't closed prematurely, I haven't addressed (3) yet (see my question above).

**Comment by ppxasjsm** [comment]
Yes sorry. I hadn't seen that third part. 

I am not sure if this is possible in practice for relative calculations, unless you write your own software? I think standard implementations in Amber, Gromacs and Sire deal with this. Maybe someone else has thoughts on this?

**Comment by ppxasjsm** [comment]
So...this third point? I also thought this was dealt with and would not pose massive issues. @mrshirts?

**Comment by mrshirts** [comment]
So, the offending section seems to be: 

"Note that, when $A$ and $B$ have different number of atoms, Eq.~\ref{eq:delta-delta-G-alchemical} implies the presence of a factor having units of volume entering both logarithms, which require unitless arguments.
However, the value of this factor is inconsequential as it cancels out in practice."

Looking at the equation, you have ln( Z(RB)Z/(RA)) - ln Z(B)/Z(A).  In theory, yes, ln( Z(RB)Z/(RA)) has units, but ln Z(B)/Z(A) has exactly cancelling units, as it can be rewritten as ln [Z(RB)Z(A)/Z(B)Z(RA)], which by counting the DoF on top and on the bottom, is unitless. So the text itself is confusing (I did a pass at clarifying), but I'm not sure exactly what the author wants to say here - I think it's actually always true.  I think there are a lot of issues in HOW the transformation is implemented (i.e. if you have dummy atoms the cancellations are not exact), but I don't think there's an issue per se in the equation itself.   I tried to handwave over this in the response as well.
